database/models.py

```diff
@@ -1,17 +1,26 @@
 import uuid
 
 from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy import Enum, func
+from sqlalchemy import Enum, func, String, Column
+from sqlalchemy.orm import DeclarativeBase
 
-db = SQLAlchemy()
 
-
-class Base(db.Model):
+class Base(DeclarativeBase):
```

Comment:
what's the advantage of shifting to this as base inheritance class?

Reply:
paired and discussed